chapters/05_numerical_results/03_what_if/main.tex

```diff
@@ -0,0 +1,804 @@
+        5.4989 & 5.4977 & 5.4960 & 5.4924 & 5.4844 & 5.4654 & 5.3875 \\
+        6.8232 & 6.8192 & 6.8150 & 6.8065 & 6.7871 & 6.7334 & 6.4906 \\
+        9.0298 & 9.0244 & 9.0187 & 9.0078 & 8.9827 & 8.9082 & 8.5145 \\
+        9.9996 & 9.9994 & 9.9992 & 9.9987 & 9.9972 & 9.9893 & 9.8571 \\
+        8.7740 & 8.8006 & 8.8249 & 8.8660 & 8.9438 & 9.1295 & 9.7157
+    \end{bmatrix}
+\end{equation*}
+
+\begin{equation*}
+    B = 
+    \begin{bmatrix}
+        1.7127 & 2.5822 & 4.6186 & 6.8497 & 8.9418 & 9.9999 & 8.2148 \\
+        1.7127 & 2.5477 & 4.5634 & 6.8047 & 8.9150 & 9.9996 & 8.3358 \\
+        1.7127 & 2.4528 & 4.3784 & 6.6441 & 8.8278 & 9.9965 & 8.5306 \\
+        1.7127 & 2.4141 & 4.2867 & 6.5470 & 8.7656 & 9.9919 & 8.6745 \\
+        1.7127 & 2.3415 & 4.0998 & 6.3265 & 8.6058 & 9.9716 & 8.9634 \\
+        1.7127 & 2.1269 & 3.4930 & 5.4885 & 7.8353 & 9.7075 & 9.7322
+    \end{bmatrix}
+\end{equation*}
+
+\begin{equation}
+    R = 
+    \begin{bmatrix}
+        0.22 & 0.06 & 0.05 & 0.05 & 0.04 & 0.03 & 0.01 \\
+        0.95 & 0.6  & 0.47 & 0.37 & 0.28 & 0.2  & 0.11 \\
+        0.97 & 0.81 & 0.72 & 0.62 & 0.51 & 0.37 & 0.21 \\
+        0.97 & 0.85 & 0.77 & 0.68 & 0.57 & 0.44 & 0.26 \\
+        0.98 & 0.89 & 0.83 & 0.76 & 0.66 & 0.53 & 0.35 \\
+        1.   & 0.95 & 0.91 & 0.87 & 0.8  & 0.7  & 0.52
+    \end{bmatrix}
+\end{equation}
+\normalsize
+
+Matrices \(A\) and \(B\) are the payoff matrices of the two hospitals and
+can be used to get the Nash equilibrium of the game.
+Using the Lemke-Howson algorithm, the following pure strategy Nash equilibrium
+is found:
+
+\begin{equation}
+    \sigma^A = (0, 0, 0, 0, 1, 0), \quad
+    \sigma^B = (0, 0, 0, 0, 0, 1, 0)
+\end{equation}
+
+The output of the Lemke-Howson algorithm indicates that the game is at a Nash
+equilibrium when hospital \(A\) plays a strategy of \(T^A = 5\) and hospital
+\(B\) plays a strategy of \(T^B = 6\).
+In fact, this is the only Nash equilibrium of the game.
+A similar outcome is obtained when using a learning algorithm.
+Figure~\ref{fig:asymmetric_replicator_dynamics_example_2} shows the output of
+the run of the asymmetric replicator dynamics algorithm.
+
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=\linewidth]{chapters/05_numerical_results/Bin/example_2/base_case.pdf}
+    \caption{Example 2: Asymmetric replicator dynamics}
+    \label{fig:asymmetric_replicator_dynamics_example_2}
+\end{figure}
+
+Asymmetric replicator dynamics converges to the same strategy as the Lemke-
+Howson algorithm, which is the pair of strategies \((T^A = 5, T^B = 6)\).
+
+Consider now the concept of the Price of Anarchy
+(\(PoA\))~\cite{roughgarden2005selfish}.
+\(PoA\) is a measure in game theory that is used to quantify the efficiency of
+the outcome of a game when players behave in a selfish way.
+More specifically, the \(PoA\) measures the ratio between the worst possible
+outcome of a game and the best possible outcome of the game.
```

Comment:
```suggestion
More specifically, the \(PoA\) measures the ratio between the worst possible
equilibria outcome of a game (so that no players have an incentive to deviate) and the best possible centrally controlled outcome of the game (the best possible collective situation).
```